mautic/mautic
# Delete expired user tokens with maintenance cleanup command

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
There is no feature which deletes expired user tokens.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
None.

#### Steps to test this PR:
1. You have to create some record in `user_tokens` table with past and future expiration date.
2. `app/console mautic:maintenance:cleanup --dry-run`
3. `app/console mautic:maintenance:cleanup` 
4. Check table content

#### List deprecations along with the new alternative:
None.

#### List backwards compatibility breaks:
None.
